### PR TITLE
Handle Twitter exception.

### PR DESCRIPTION
**Pull request by ahmubashshir**
Handle Twitter exception and show User the message returned by api.

### CODE REVIEW

File: hooks/tweet.py [hunk 1]
```diff
@@ -18,26 +18,33 @@
...
         access_token_secret=ACCESS_SECRET)
 
 def status_changed(engine, show, old_status):
     api_name        = engine.api_info['name']
-    finished_status = engine.mediainfo['status_finish']
+    finished_status = engine.mediainfo['statuses_finish']
     score_max       = engine.mediainfo['score_max']
 
-    if show['my_status'] == finished_status:
+    if show['my_status'] == finished_status[0] if finished_status is list else finished_status:
```
Comment (FichteFoll) [inline hooks/tweet.py]:
```suggestion
    if show['my_status'] == finished_status[0] if isinstance(finished_status, list) else finished_status:
```

**Comment by FichteFoll** [comment]
You should also use whitespace more, i.e. in line 47:
```py
engine.msg.warn('Twitter', "[%d] %s"%(e.message[0]['code'],e.message[0]['message']))
```

**Comment by ahmubashshir** [comment]
Can it be merged now?

**Comment by ahmubashshir** [comment]
I also didn't.I just added an exception handler that would notify user about the error.

**Pull request merged**